DOCSP-40542 Compass Quit Functionality Update (#640)
mongodb/docs-compass: source/settings/settings-reference.txt

       enabled. You cannot toggle this option on Linux. You can toggle this 
       setting only on Windows or macOS.

   * - Show Quit Confirmation Dialog
     - General

>      - Toggle whether a confirmation dialog is shown when quitting Compass.

Reviewer: (term and active voice)
I'm a little concerned about toggle. I think we could get more specific. Also we want to stay away from "is shown". So maybe something like:
Specifies whether to display the confirmation dialog when you exit Compass.

       Checking the ``Do not ask again`` checkbox on the quit dialog 

>        disables this setting.

Reviewer: (structure)
To disable this setting, check "Do not ask again".

Author: This removes the reference to the quit dialog box which is a separate element than where the setting is stored. This is a supplemental way to turn off the setting option. 

So it needs to be:

"You can also check the ``Do not ask again`` checkbox on the quit dialog to disable the setting."

>        You can also toggle the quit dialog with the keyboard shortcut:
>        :kbd:`ctrl` + :kbd:`Q` on Windows or :kbd:`cmd` + :kbd:`Q` on Mac.

Reviewer 2: Maybe:
```suggestion
       The quit dialog is also shown when quitting with the keyboard shortcut:
       :kbd:`ctrl` + :kbd:`Q` on Windows or :kbd:`cmd` + :kbd:`Q` on Mac.
```
